.editorconfig

```diff
@@ -267,8 +267,7 @@ dotnet_diagnostic.CA1711.severity = suggestion
 dotnet_diagnostic.CA1309.severity = suggestion
 
 # CA1725: Parameter names should match base declaration
-# TODO: Enable this rule as warning and fix violations.
-dotnet_diagnostic.CA1725.severity = suggestion
+dotnet_diagnostic.CA1725.severity = warning
```

Comment:
This setting is redundant. We set all CA rules to be warning by default here: https://github.com/dotnet/roslyn-analyzers/blob/edf95f72a20ed1c090a1639c8ec85bb5854fe09e/src/Directory.Build.props#L18-L22